### PR TITLE
Wherescape

### PR DESCRIPTION
**Pull request by david-wirelab**

### CODE REVIEW

File: packer.json [hunk 1]
```diff
@@ -12,11 +12,11 @@
       "source_ami_filter": {
         "filters": {
           "virtualization-type": "hvm",
-          "name": "Windows_Server-2012-R2_RTM-English-64Bit-Base-*",
+          "name": "base-win-2012-wherescape-*",
           "root-device-type": "ebs"
         },
         "owners": [
-          "801119661308"
+          "093401982388"
```
Comment (chrisns) [inline packer.json:19]: you probably want prod+notprod in this list

**Review by chrisns — CHANGES_REQUESTED** [review]

File: packer.json (hunk 1 above)
Comment (david-wirelab) [reply packer.json:19]: Prod+notprod will not use this directly but the encrypted copy right?

Comment (chrisns) [reply packer.json:19]: you need to share the ami with prod+notprod so that it can be copied there

Comment (david-wirelab) [reply packer.json:19]: it is shared at the ami_users section. Owners refers to the owner of the AMI which is CI.

      "ami_users": [
        "483846886818",
        "337779336338"
      ]

Comment (chrisns) [reply packer.json:19]: Yes. Sorry. You’re quite right

**Review by chrisns — APPROVED** [review]

**Pull request merged**